spring-beans/src/main/java/org/springframework/beans/factory/support/DefaultSingletonBeanRegistry.java

```diff
@@ -16,7 +16,6 @@
 
 package org.springframework.beans.factory.support;
 
-import java.util.Collections;
```

Comment:
Surprise: I just noticed that 902e5707a88522a158985138e4ec8b1061184d1c introduced additional usage of `java.util.Collections`, so the code no longer compiles when merged into `main`.

Though fret not: I'll add the import back locally. 😉